packages/twenty-server/src/database/typeorm/metadata/migrations/1711124466598-addRemoteServerTable.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddRemoteServerTable1711124466598 implements MigrationInterface {
+  name = 'AddRemoteServerTable1711124466598';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "metadata"."remoteServer" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "fdwId" uuid NOT NULL DEFAULT uuid_generate_v4(), "fdwType" character varying, "fdwOptions" jsonb, "userMappingOptions" jsonb, "workspaceId" uuid NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "PK_8e5d208498fa2c9710bb934023a" PRIMARY KEY ("id"))`,
+    );
```

Comment:
@FelixMalfait FYI impossible currently to add a foreign key constraint on the workspace. Both entities live in separated schemas, and our type orm setup makes only accessible the entities that live in the same one.

There are probably possibilities to update this config but I lost too much time on it. And what we do currently is only adding the id without foreign key when metadata and core have a relation (see objectmetadata for example)